lib/utils.pm

```diff
@@ -252,6 +253,11 @@ sub get_netboot_mirror {
     return get_var('MIRROR_' . uc($m_protocol));
 }
 
+# Device order in QEMU is switched - bsc#1038275
+sub get_extra_disk() {
+    return check_var('ARCH', 'ppc64le') ? '/dev/vda' : '/dev/vdb';
```

Comment:
* can we identify the disks by label or id or by reading out something from /etc/fstab? It's unsafe to use hardcoded '/dev/vda' and such
* Also, this was suggested in the bug itself
* Also @sysrich changed the qemu version on the ppc64le workers again to 2.9 which should make drive order in line with what the have for x86_64
* Also we have the same problem for aarch64
* Also it's not an ARCH-related problem, rather MACHINE (if it all, see above)?